### DISCUSSION TITLE
Nextcloud 27.1.3 Maintenance mode is not working properly

### DISCUSSION THREAD

**Discussion by pali7x** [nextcloud/all-in-one#3637] [Questions] [answered]
### Steps to reproduce
1. occ maintenance:mode --on
2. Access via web

### Expected behavior 
Supposedly under maintenance mode, accessing web nextcloud will show its under maintenance mode.

### Actual behavior 
It did not happen. I can still browsing around via web, however any occ command such as files:scan won't work as expected under maintenance mode on.

### Host OS 
Debian 11, nginx, php8.2

#### Nextcloud AIO version 
27.1.3

#### Current channel 
Stable

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/documentation/pull/10645/files

**Reply by pali7x** [reply]
That's it. Thanks a lot, i forgot that i need to restart the php due to my config.
